bin/ch-tar2dir

```diff
@@ -9,58 +9,58 @@ Unpack a Docker export tarball into a directory.
 
 Usage:
 
-  $ $(basename "$0") TARBALL DIR
+  $ $(basename "$0") tarball DIR
 EOF
 )
 
 parse_basic_args "$@"
 
 if [ "$1" = --verbose ]; then
-    VERBOSE=v
+    verbose=v
```

Comment:
Change this to `verbose=yes` to match style elsewhere. (Note: boolean variables in shells scripts are tricky. We use the convention empty/unset=false, "yes"=true.)